sql/core/src/test/scala/org/apache/spark/sql/TPCDSBase.scala

```diff
@@ -36,6 +36,12 @@ trait TPCDSBase extends SharedSparkSession with TPCDSSchema {
     "q81", "q82", "q83", "q84", "q85", "q86", "q87", "q88", "q89", "q90",
     "q91", "q92", "q93", "q94", "q95", "q96", "q97", "q98", "q99")
 
+  // Since `tpcdsQueriesV2_7_0` has almost the same queries with these oens below,
+  // we skip them in the TPCDS-related tests.
```

Comment:
Well, this hides the previous reasoning. I believe it's worth to keep SPARK-35327 comment explicitly.
```
SPARK-35327: Filters out the TPC-DS queries that can cause flaky test results
```

Reply:
Sure, udpated.